mangoPURE/data/transforms.py

```diff
@@ -0,0 +1,280 @@
+from datasets import Dataset as HfDataset
+from .base import MixedExample, SegmentInfo, MixerTransform
+import random
+import torch
+from .utils import expand_tensor, calc_audio_adjustment_coef
+
+
+class CreateRandomBlankAudio(MixerTransform):
+    def __init__(
+            self,
+            min_secs: float = 10.0,
+            max_secs: float = 20.0,
+            rate: int = 16000
```

Comment:
it would be convenient to have global SR parameter. something like mango.MAIN_SR=16_000 and not pass it to every function